docs/introduction/docker-stream-logging.md

```diff
@@ -0,0 +1,104 @@
+# Logging with docker containers
+If there is need to debug some problem in application that is deployed within many docker containers,
+it is better to have logs from different sources centralized.
+For docker, there is easy way how to do it by sending all logs into streams `STDOUT` and `STDERR`.
+
+## Php-fpm
+In php-fpm docker conainer configuration is set by default that php-fpm service sends error and access log messages into `STDERR` stream.
+`/usr/local/etc/php-fpm.d/docker.conf`
+```
+[global]
+error_log = /proc/self/fd/2
+
+[www]
+; if we send this to /proc/self/fd/1, it never appears
+access.log = /proc/self/fd/2
+
+clear_env = no
+
+; Ensure worker stdout and stderr are sent to the main error log.
+catch_workers_output = yes
+```
+For other docker containers we can easily setup logging of any service inside container like it is for php-fpm.
```

Comment:
I don't understand. First, you say that php-fpm is set by default -I dont see where I can change the configuration of this container. Here you say that we can easily setup logging - how, how we can set up logging? Where? If I would need to set up configurations for logging into the stream I do not know what I should do.